website/home_page.html

```diff
@@ -0,0 +1,221 @@
+<!doctype html>
```

Comment:
The home page is really beginning to get good shape. This is from a very high level what I envisioned.

However, as it is it really looks unfinished with the rest of the website. I don't like the green monotone coloring at all; it'd be nice if the background was some kind of texture; maybe a low resolution image of a forrest or something.

Even though the layout is great, the website looks very unfinished to be honest. Things are looking very square., the coloring is uneven, and there's no style at all to it. I would again suggest going over the big collection of websites from 8VC and mimicking what they did; those websites look super catchy and that's what ours would ideally look like.